### ISSUE TITLE
Why fix gamma in the BatchNorm?

### ISSUE THREAD

**Issue by twmht** [deepinsight/insightface#344]
Hi,

I saw gamma of the batch normalization after the fully connection is fixed during training.

https://github.com/deepinsight/insightface/blob/master/src/symbols/symbol_utils.py#L33

I don't understand why you did this.

Any math behind that?

thank you

**Comment by nttstar** [comment]
From experiments.

**Comment by twmht** [comment]
@nttstar 

我直接用中文問好了

網路架構可調的參數有好幾百個甚至上千個，請問你們是怎麼想到要去試著修改這個參數？

**Comment by dl8207531** [comment]
所有输出层bn都是fixed gamma 我理解是输出层是可以线性化的 所以不需要乘上gamma

**Comment by twmht** [comment]
@dl8207531 

我覺得不是，因為除了最後一層以外的  BN， gamma 都是有在學習的。
